src/components/navigation/navigation.component.html

```diff
@@ -0,0 +1,7 @@
+<nav class="tree" role="navigation">
+    <ol class="nav">
+        <ng-content></ng-content>
+    </ol>
+</nav>
+
+<router-outlet></router-outlet>
```

Comment:
Isn't it up to the application to determine where the router outlet goes? Why was this added here?